Dashboard/app/js/lib/views/devices/assignment-edit-views.jsx

```diff
@@ -591,15 +594,20 @@ FLOW.AssignmentEditView = FLOW.ReactComponentView.extend(
         const datapointAssignment = this.map(item => ({
           id: item.get('keyId'),
           deviceId: item.get('deviceId'),
-          datapoints: item.get('dataPointIds'),
+          datapointIds: item.get('dataPointIds'),
+          datapoints: [],
+
+          // pagination details
+          hasMoreRawIds: false,
+          currentDatapointsIdsChunk: 0,
```

Comment:
A chunk to me suggest an array rather than the index of an item in an array. maybe rename to better reflect that this is an array item index?